doc/Makefile.am

```diff
@@ -0,0 +1,160 @@
+# This Makefile started with the default Makefile produced by the Sphinx
+# initialization process, which we then modified over time. During the
+# Automake-ification, I stripped out most of the boilderplate and left only
+# the targets that we use.
+
+EXTRA_DIST = \
+bugs.rst \
+charliecloud.rst \
+ch-build2dir_desc.rst \
+ch-build2dir.rst \
+ch-build_desc.rst \
+ch-builder2squash_desc.rst \
+ch-builder2squash.rst \
+ch-builder2tar_desc.rst \
+ch-builder2tar.rst \
+ch-build.rst \
+ch-checkns_desc.rst \
+ch-checkns.rst \
+ch-dir2squash_desc.rst \
+ch-dir2squash.rst \
+ch-fromhost_desc.rst \
+ch-fromhost.rst \
+ch-grow_desc.rst \
+ch-grow.rst \
+ch-mount_desc.rst \
+ch-mount.rst \
+ch-pull2dir_desc.rst \
+ch-pull2dir.rst \
+ch-pull2tar_desc.rst \
+ch-pull2tar.rst \
+ch-run_desc.rst \
+ch-run-oci_desc.rst \
+ch-run-oci.rst \
+ch-run.rst \
+ch-ssh_desc.rst \
+ch-ssh.rst \
+ch-tar2dir_desc.rst \
+ch-tar2dir.rst \
+ch-tar2squash_desc.rst \
+ch-tar2squash.rst \
+ch-test_desc.rst \
+ch-test.rst \
+ch-umount_desc.rst \
+ch-umount.rst \
+command-usage.rst \
+conf.py \
+dev.rst \
+faq.rst \
+favicon.ico \
+index.rst \
+install.rst \
+logo-sidebar.png \
+make-deps-overview \
+rd100-winner.png \
+see_also.rst \
+tutorial.rst
+
+if ENABLE_MAN
+man1_MANS = \
+man/charliecloud.1 \
+man/ch-build.1 \
+man/ch-build2dir.1 \
+man/ch-builder2squash.1 \
+man/ch-builder2tar.1 \
+man/ch-checkns.1 \
+man/ch-dir2squash.1 \
+man/ch-fromhost.1 \
+man/ch-grow.1 \
+man/ch-mount.1 \
+man/ch-pull2dir.1 \
+man/ch-pull2tar.1 \
+man/ch-run.1 \
+man/ch-run-oci.1 \
+man/ch-ssh.1 \
+man/ch-tar2dir.1 \
+man/ch-tar2squash.1 \
+man/ch-test.1 \
+man/ch-umount.1
+endif
+
+if ENABLE_HTML
+nobase_html_DATA = \
+html/searchindex.js \
+html/_images/rd100-winner.png \
+html/command-usage.html \
+html/dev.html \
+html/faq.html \
+html/index.html \
+html/install.html \
+html/tutorial.html
+endif
+
+
+# NOTE: ./html might be a Git checkout to support "make web", so make sure not
+# to delete it.
+CLEANFILES = $(man1_MANS) $(html_DATA) _deps.rst
+if ENABLE_HTML
+CLEANFILES += $(nobase_html_DATA) html/.buildinfo html/.nojekyll
+# Automake can't remove directories.
+clean-local:
+	rm -Rf doctrees html/_static
+endif
+
+# Automake can't install and uninstall directories. _static contains around
+# one hundred files in several directories, and I'm pretty sure the contents
+# change depending on Sphinx version and other details, so we can't just list
+# the files. These targets deal with it as an opaque directory.
+if ENABLE_HTML
+install-data-hook:
+	cp -r html/_static $(DESTDIR)$(htmldir)/html
+	find $(DESTDIR)$(htmldir)/html/_static -xtype d -exec chmod 755 {} \;
+	find $(DESTDIR)$(htmldir)/html/_static -xtype f -exec chmod 644 {} \;
+uninstall-hook:
+	   test -d $(DESTDIR)$(htmldir)/html/_static \
+	&& rm -Rf $(DESTDIR)$(htmldir)/html/_static
+endif
```

Comment:
The downside is that `html` then becomes non-independent. This would complicate `make web`, and would require us to handle differing Sphinx theme locations. On my Debian box with Sphinx installed via `pip3`, this is `/usr/local/lib/python3.5/dist-packages/sphinx_rtd_theme/static`.

My weak take is that it's better to deal with it in the spec file as above, but I could be convinced otherwise. Opinions?